ci.rb

```diff
@@ -329,7 +329,9 @@ def get_limits(t_options, t_client, t_repo)
               end
 
               if compiler[:collect_performance_results]
-                p.collect_performance_results
+                p.collect_file_sizes
+                p.collect_perf_results
+                p.collect_valgrind_counters_results
```

Comment:
OK, so doing a little more stuff for configurations that have `collect_performance_results` enabled.  Cool.